lib/galaxy/managers/citations.py

```diff
@@ -47,7 +47,7 @@ def _raw_get_bibtex(self, doi):
         doi_url = "https://doi.org/" + doi
         headers = {'Accept': 'text/bibliography; style=bibtex, application/x-bibtex'}
         req = requests.get(doi_url, headers=headers)
-        return req.text
+        return req.content.decode("utf-8")
```

Comment:
that should be exactly the same as `req.text`:
```

In [1]: url = 'https://usegalaxy.org/api/version'

In [2]: import requests

In [3]: r = requests.get(url)

In [4]: r.text
Out[4]: '{"version_major": "20.01", "extra": {}}'

In [5]: r.content
Out[5]: b'{"version_major": "20.01", "extra": {}}'

In [6]: r.content.decode('utf-8')
Out[6]: '{"version_major": "20.01", "extra": {}}'
```

Reply:
Try 
```
import requests
doi_url = "https://doi.org/10.1093/bioinformatics/bts252"
headers = {'Accept': 'text/bibliography; style=bibtex, application/x-bibtex'}
req = requests.get(doi_url, headers=headers)
print(req.content.decode("utf-8"))
```
and 
```
import requests
doi_url = "https://doi.org/10.1093/bioinformatics/bts252"
headers = {'Accept': 'text/bibliography; style=bibtex, application/x-bibtex'}
req = requests.get(doi_url, headers=headers)
print(req.text)
```
You will see encoding problems

Reply:
It seems the issue is that `req.encoding` is 'ISO-8859-1', while it actually is 'utf-8'.

Reply:
@nsoranzo 
Ok, should I change it to 
```
req = requests.get(doi_url, headers=headers)
req.encoding = req.apparent_encoding
```
does it look bettter?